core/src/main/java/org/apache/calcite/sql/type/SqlTypeFamily.java

```diff
@@ -244,7 +244,7 @@ public Collection<SqlTypeName> getTypeNames() {
     case EXACT_NUMERIC:
       return SqlTypeUtil.getMaxPrecisionScaleDecimal(factory);
     case INTEGER:
-      return factory.createSqlType(SqlTypeName.BIGINT);
+      return factory.createSqlType(SqlTypeName.INTEGER);
```

Comment:
Why this change? I would think that `BIGINT` is a good default concrete type for the family `INTEGER`, since it's the least restrictive in the family. Seems like this may have unwanted effects, if it causes inference for the `INTEGER` family to use a too-narrow type.

Reply:
yep, thanks, seems it redundant change